src/ui/profile_preferences.ui

```diff
@@ -2219,21 +2232,38 @@ you can use it but there could be issues with aligning columns of text</string>
             </property>
            </widget>
           </item>
-          <item row="1" column="0" colspan="4">
+          <item row="1" column="0">
+           <widget class="QLabel" name="label_4">
+            <property name="text">
+             <string>Room border color:</string>
+            </property>
+           </widget>
+          </item>
+          <item row="1" column="1">
+           <widget class="QPushButton" name="pushButton_room_border_color">
```

Comment:
*I* like to use "camelCasing" for the entity type and the what it is - but keep a `_` as a divider between them - also the `snake_case` "room_border_color" is marginally longer than "roomBorderColor" so may I suggest:

```suggestion
           <widget class="QPushButton" name="pushButton_roomBorderColor">
```

Reply:
Yeah, there is a lot of different conventions in same file. Took that one from `pushButton_foreground_color_2`.